README.md

```diff
@@ -2,9 +2,34 @@
 
 A Javascript client library for [Ably Realtime](https://www.ably.io), a realtime data delivery platform.
 
+## Supported platforms
+ 
+ This SDK is compatible with the following platforms:
+ 
+| Node.js  | Browsers           | 
+|----------|--------------------|
+|   4      | IE 8+              |
+|   6      | Chrome 64+         |
+|   8      | Firefox 58+        |
+|   9      | Safari macOS 11    |
+|          | Safari iOS 11      |
+|          | Chrome on Android 6|
```

Comment:
This seems overly restrictive -- I think we're underselling ourselves.

E.g. it makes it sound like we don't support firefox <=57, which may be rather off-putting for someone on, say, the current extended-support release (52). Or safari on iOS 10, or Chrome on Android 8, or the default browser on a Samsung phone, etc. etc. etc.

In reality we we do support all of those, in the sense that if someone found a bug in one of them we would put effort into reproducing and fixing it. We just don't test them on ci. But that's just for practical reasons -- if we could test every one of the thousands of platforms we consider 'supported' in that sense we would; since we can't, we test a selection of common ones. But that doesn't mean the sdk isn't compatible with others.

ISTM what's relevant to the customer is what we support, in that sense (what we'll accept and fix bug reports on), not what we happen to test in ci (which we reserve the right to change at any point without notice). So I'd suggest making the list of what we support be that. WDYT?

Reply:
@SimonWoolf I have a similar concern.
We still have to extra time on the CI (before we max out the allowed session length). So we could add some older browsers like: 

- Firefox 52
- Chrome TDB
- Safari iOS10
- Chrome on Android 4

Reply:
sure, we can discuss that in the CI issue. my main point for the purposes of this PR is that the libraries we support is a (large) superset of the libraries we test in ci, and we shouldn't imply otherwise.

Reply:
Isn't that addressed with my comments re fallbacks?

Reply:
I don't think so. ISTM that having a big table at the top saying "we're only compatible with firefox 58+, chrome on Android 4.0, ..." just because those are the versions we currently test in ci is just misleading in a way that isn't really anything to do with whether we have transport fallbacks (it's not like firefox 57 doesn't support websockets). Rather it's to do with whether we'll fix bugs reported against firefox 57. Which we will, even through we don't have that specific version in ci. No?

Reply:
I agree with @SimonWoolf's comments here.  A visitor is probably not going to read the detail in reality, they are going to scan, see a table, and assume we don't support Opera or old versions, when we do.